tests/testthat/test-download_model.R

```diff
@@ -0,0 +1,20 @@
+httptest2::with_mock_dir("download-model-success", {
```

Comment:
None of these tests need `skip_if_offline()` because they are mocking HTTP requests.